style.css

```diff
@@ -341,3 +369,124 @@ textarea {
   color: #42526e;
   line-height: 32px;
 }
+
+@media screen and (min-width: 992px) {
+  #headline {
+    height: 145vh;
+    background-image: url('./assets/header-llustration.svg');
+  }
+
+  .navbar {
+    justify-content: center;
+    gap: 32px;
+  }
+
+  .topNavigation {
+    display: block;
+    text-decoration: none;
+    color: #344563;
+    font-weight: 600;
+    font-size: 15px;
+  }
+
+  .menuIcon {
+    display: none;
+  }
+
+  .welcome {
+    display: none;
+  }
+
+  .titleContainer h1 {
+    font-size: 48px;
+  }
+
+  .titleContainer h3 {
+    font-size: 48px;
+    width: 100%;
+  }
+
+  .headlineDescription p {
+    width: 65%;
+  }
+
+  .headlineContainer {
+    margin-top: 0;
+    margin-bottom: 200px;
+  }
+
+  .arrowBtn {
+    margin-top: 60px;
+  }
+
+  .divider {
+    display: none;
+  }
+
+  .blueTitleText {
+    width: 100%;
+  }
+
+  .grid-card-container {
+    grid-template-rows: auto;
+    grid-template-columns: 1fr 1fr 1fr;
+    column-gap: 24px;
+  }
+
+  .project-card .projectImage {
+    flex: 1 0 45%;
+  }
+
+  .aboutmeDescription {
+    width: 60%;
+  }
+
+  #contact {
+    display: flex;
+    padding-left: 5%;
+    padding-right: 5%;
+    padding-bottom: 25vh;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin: auto;
+    margin-top: 20vh;
+    background-image: url('./assets/ilustration-left-contact.svg'), url('./assets/ilustration-contact.svg'), url('./assets/ilustration-right-contact.svg');
+    background-repeat: no-repeat, no-repeat, no-repeat;
+    background-position: left bottom, 25% 65%, right;
+  }
+
+  #contact h3 {
+    flex: 50%;
+    text-align: justify;
```

Comment:
- [ ] OPTIONAL: I don't think this `text-align: justify` is a good styling for the text here. The original design looks like `text-align: left`. Please compare the attached screenshot images of yours and original design. 

![image](https://user-images.githubusercontent.com/64894927/121435634-ebdf9b00-c987-11eb-8d47-39ce3030079e.png)   
![image](https://user-images.githubusercontent.com/64894927/121435676-fb5ee400-c987-11eb-8dc2-999761aa969f.png)